protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/toconnectschema/ProtobufSchemaToConnectSchemaConverter.java

```diff
@@ -116,8 +120,15 @@ private SchemaBuilder toConnectSchemaBuilderForField(final Descriptors.FieldDesc
 
         //Protobuf provides different types of integers.
         //We add metadata to Connect schema to store the original type used.
+        //final SchemaBuilder schemaBuilder = new SchemaBuilder(connectType);
```

Comment:
Why is this commented? Should it be deleted?

Reply:
hmm - I think I wrote it this way before you refactored this class and defined a schemabuilder at the top, and commented it out when I first merged. Since it is working now, I will delete it